build.gradle.kts

```diff
@@ -497,8 +506,25 @@ project(":debugger") {
     }
     dependencies {
         implementation(project(":"))
+        antlr("org.antlr:antlr4:4.10.1")
+        implementation("org.antlr:antlr4-runtime:4.10.1")
         testImplementation(project(":", "testOutput"))
     }
+    tasks {
+        withType<KotlinCompile> {
+            dependsOn(":debugger:generateGrammarSource")
+        }
+
+        generateGrammarSource {
+            arguments.add("-no-listener")
```

Comment:
@Undin Note, I have just added this option to skip listener generation since the ANTLR listeners are not used in the current implementation